packages/x-date-pickers/src/DesktopDatePicker/tests/DesktopDatePicker.test.tsx

```diff
@@ -24,39 +24,39 @@ describe('<DesktopDatePicker />', () => {
         />,
       );
 
-      fireEvent.click(screen.getByLabelText(/switch to year view/i));
+      // Parent element is used to avoid the ripple effect triggering act warnings.
+      await user.click(screen.getByLabelText(/switch to year view/i).parentElement!);
```

Comment:
This was annoying to figure out, not sure there is a way to disable the ripple.

Reply:
Maybe this would be cleaner? 🤔 
```suggestion
      // Specifically selecting the button element to avoid the ripple effect triggering act warnings.
      await user.click(screen.getByLabelText(/switch to year view/i, { selector: 'button' }));
```

Reply:
Didn't work https://app.circleci.com/pipelines/github/mui/mui-x/89406/workflows/8563c3ec-4ae3-4016-a265-04fa3855103f/jobs/517815